src/frequenz/channels/_bidirectional.py

```diff
@@ -117,21 +122,29 @@ def __init__(self, client_id: str, service_id: str) -> None:
             service_id: A name for the service end of the channels.
         """
         self._client_id = client_id
+        """The name for the client, used to name the channels."""
```

Comment:
This is really confusing, but I know the original docs for `client_id` is and even the name of the variable.

But I guess we shouldn't fix the documentation, but fix the interface instead:

* #187